src/app/index.js

```diff
@@ -24,7 +24,7 @@ function findTest(test, filename){
     }
 }
 
-function loadTests(data, autorun, notifyLoaderFunction){
+function loadTests(data, autorun, fileSync, notifyLoaderFunction){
```

Comment:
I think new argument should be appended, since `notifyLoaderFunction` may using by some one.